### ISSUE TITLE
linter: `jsx-a11y/iframe-has-title` has false positive on `title={a ?? b}`

### ISSUE THREAD

**Issue by stefanprobst** [oxc-project/oxc#22422]
### What version of Oxlint are you using?

1.64.0

### What does your `.oxlintrc.json` (or `oxlint.config.ts`) config file look like?

```jsonc
{
  "plugins": ["jsx-a11y"],
  "categories": {
    "correctness": "off"
  },
  "rules": {
    "jsx-a11y/iframe-has-title": "error",
  },
}
```

### What happened?

oxlint reports error, even though `title` prop is provided on `<iframe>`:

see reproduction at https://github.com/stefanprobst/issue-oxlint-jsx-iframe
